cfme/tests/test_soap.py

```diff
@@ -460,6 +460,10 @@ def test_vm_tagging(self, soap_client):
 
 
 class TestProvisioning(object):
+    WAIT_TIME = 300
+    WAIT_TIME_SLOW = 600
+    SLOW_PROVIDERS = {"rhevm", "scvmm"}
+
```

Comment:
Crap, I was unclear. Sorry about that. I liked the comments, just not that they were inline.